README.md

```diff
@@ -53,6 +53,7 @@
 - [Brackets](http://brackets.io/) - A modern text editor that understands web design. ![Open-Source Software][OSS Icon]
 - [Light Table](https://github.com/LightTable/LightTable) - A customizable editor with REPL support for Clojure, JavaScript, Python and others. ![Open-Source Software][OSS Icon]
 - [Macvim](https://github.com/b4winckler/macvim) - Vim, the text editor. ![Open-Source Software][OSS Icon]
+- [VimR](http://vimr.org} - Vim, redefined. ![Open-Source Software][OSS Icon]
```

Comment:
Also I think the description should be  - `Vim, refined.` referring to their website.